_posts/2016/01/2016-01-14-election-jason-williams.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+author: "Jason Williams"
+title: "2016 Election: Jason Williams"
+date: 2016-01-14
+time: "15:00:00"
+category: ["Software Carpentry Foundation"]
```

Comment:
Please add "Election 2016" as a category as well (or I can do it once this is checked in).